packages/main/src/themes/ValueStateMessage.css

```diff
@@ -87,6 +90,71 @@
 }
 
 .ui5-valuestatemessage-popover::part(header),
-.ui5-suggestions-popover-with-value-state-header::part(header) {
+.ui5-popover-with-value-state-header::part(header),
+.ui5-popover-with-value-state-header-phone::part(header) {
 	margin-bottom: 0;
 }
+
+.ui5-popover-with-value-state-header .ui5-valuestatemessage-root {
+	padding: var(--_ui5_value_state_message_padding);
```

Comment:
You are right. This line is redundant for .ui5-popover-with-value-state-header .ui5-valuestatemessage-root, because the padding is set for .ui5-valuestatemessage-root. I will remove it.

The next selector is needed because it sets different than the default padding for mobile devices